http/swagger.yml

```diff
@@ -4720,54 +4720,57 @@ paths:
           content:
             application/json:
               schema:
-                type: array
-                items:
-                  type: object
-                  properties:
-                    id:
-                      type: string
-                    orgID:
-                      type: string
-                    name:
-                      type: string
-                    description:
-                      type: string
-                    sources:
-                      type: array
-                      items:
-                        type: string
-                    urls:
-                      type: array
-                      items:
-                        type: string
-                    createdAt:
-                      type: string
-                      format: date-time
-                      readOnly: true
-                    updatedAt:
-                      type: string
-                      format: date-time
-                      readOnly: true
-                    resources:
+                type: object
+                properties:
+                  stacks:
```

Comment:
This looks great!


![check](https://media.tenor.com/images/cbae2dfd31aa5ec2fcb7f46b65e1550f/tenor.gif)